### PR TITLE
Guillaume Doyen's Earth PR

### PR DESCRIPTION
**Pull request by GuillaumeDoy**
Here is my first PR. I just added a picture of the earth.

### CODE REVIEW

**Comment by MKourdourli** [comment]
I approve this Earth image!

File: guillaume-doyen_test-file.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+?MMMMMMMMP                  9MMMMMMMM}       -
+-?MMMMMMM                  |MMMMMMMMM?,d-    '
+ :|MMMMMM-                 `MMMMMMMT .M|.   :
+  .9MMM[                    &MMMMM*' `'    .
+   :9MMk                    `MMM#"        -
+     &M}                     `          .-
+      `&.                             .
+        `~,   .                     ./
+            . _                  .-
+              '`--._,dd###pp=""'
```
Comment (PabloRdrRbl) [inline guillaume-doyen_test-file.txt:23]: For example, I could ask you for a modification in this line before accepting your review. But I won't be mean :)

**Review by ivalaginja — APPROVED** [review]
Whoa! Excellent addition!

**Comment by GuillaumeDoy** [comment]
> 
> 
> I approve this Earth image!

Haha thanks a lot Mehdi! Nice to see you here!

**Comment by GuillaumeDoy** [comment]
> 
> 
> Whoa! Excellent addition!

Thank you Iva!

**Review by PabloRdrRbl — APPROVED** [review]
Now, I approve.

File: guillaume-doyen_test-file.txt (hunk 1 above)
Comment (GuillaumeDoy) [reply guillaume-doyen_test-file.txt:23]: ahaha!! that's a huge modification. I disagree, I think we should save the Earth and don't make it vanishing though! ;-)

**Pull request merged**